app/views/fields/hours_field/_show.html.erb

```diff
@@ -0,0 +1 @@
+<%= link_to "Hours", root_path+"hours/"+field.to_s %>
```

Comment:
You should replace `root_path+"hours/"+field.to_s` with the route helper `library_hours_path(field.to_s)` so that if we decide to change the url path from hours to openinghours or something similar, it will be automatically updated.